### DISCUSSION TITLE
Github copilot

### DISCUSSION THREAD

**Discussion by JuanmeGarcia** [community/community#30848] [Copilot Conversations] [answered]
Hi, i am software engineer student so I got the student tier last year for all the benefits offered, I right now i don't have access to github copilot which theoretically i should have. I just wanna know if it was any kind of problem or a solution for this inconvenient. 

Thanks! 

![imagen_2022-08-25_173634820](https://user-images.githubusercontent.com/77397709/186763934-bf6a7a87-1ae6-4134-8fd0-8adc0d426e63.png)

**Comment by davecheney** [comment, marked as answer]
@JuanmeGarcia if you have an active student coupon, please ensure that coupon is active in your account then the option for student access will appear when you visit https://github.com/github-copilot/signup

**Comment by JuanmeGarcia** [comment]
Indeed, i am able to use it but it was a little bit weird receiving an
email from github telling me that i have to get a subscription. Thanks for
answering my doubt!!

**Reply by davecheney** [reply]
Thank you, I'll pass this on to the folks that manage the emails.

**Comment by cparello** [comment]
my company pays for my copilot and somethign is wrong with your billing system they have not fixed it for days

**Reply by davecheney** [reply]
I am sorry, we cannot assist you with this issue via this forum. Please contact support https://support.github.com/contact
